src/commands/applications/list.ts

```diff
@@ -133,11 +150,14 @@ use applications:get to get full details for a single app.
 `)
     }
 
-    if (totalSize > count) {
-      this.log(`\nShowing ${chalk.cyan(count)} of ${chalk.cyan(totalSize)} application${s(count)}.\n`)
-    }
-
     super.outputCLITable(transformedData, columns)
+
+    if (totalSize >= count) {
+      const resultInformation = offset + count > 1 ? `${chalk.cyan(offset + 1)}-${chalk.cyan(offset + count)}` : chalk.cyan(offset + count)
+
+      this.log()
+      this.log(`Result${s(offset + count)} ${resultInformation} of ${chalk.cyan(totalSize)} shown.`)
```

Comment:
The plural of "Result" only depends on the `count` of items present in the response.